ddtrace.gemspec

```diff
@@ -75,4 +75,15 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'datadog-ci', '~> 0.8.1'
 
   spec.extensions = ['ext/datadog_profiling_native_extension/extconf.rb', 'ext/datadog_profiling_loader/extconf.rb']
+
+  spec.post_install_message = <<-MSG
+    Thank you for installing ddtrace.
+
+    As of version 2, the `ddtrace` gem has been renamed to `datadog` to better represent the full suite of Datadog products.
+    The 1.x series will now only receive maintenance updates for security and critical bug fixes.
+
+    To upgrade, please update your Gemfile to replace `ddtrace` with `datadog`.
```

Comment:
I have a suggestion that goes in a bit of another direction from Marco's suggestion. Since as Marco pointed out very correctly "this message will be out in the wild for a long time", I was thinking we could have a very minimalist message, so that people wouldn't get very annoyed by it.

E.g. Yes I'm usually the person that would suggest a nice big message like the one we're using, but I think this situation may deserve a special consideration.

Specifically, I was thinking something like:

> The `ddtrace` gem has been renamed to `datadog`. We strongly recommend upgrading. https://dtdg.co/dd-gem

(The url shortener even allows us to point the message at a different place in the future ).

Reply:
The idea here being, you still see the message, but it's not on your face/annoying every time you run bundle install.

Reply:
I still think the short-and-sweet message with the redirect link would be the best solution here :)